pages/installing-slimefun.md

```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
```

Comment:
You actually shouldn't add this header, the wiki automatically does it, so you will have two headers now.,